Spigot-Server-Patches/0247-Allow-simple-built-in-biome-swapping.patch

```diff
@@ -0,0 +1,202 @@
++    public static final BiomeBase b;
++    public static final BiomeBase c;
++    public static final BiomeBase d;
++    public static final BiomeBase e;
++    public static final BiomeBase f;
++    public static final BiomeBase g;
++    public static final BiomeBase h;
++    public static final BiomeBase i;
++    public static final BiomeBase j;
++    public static final BiomeBase k;
++    public static final BiomeBase l;
++    public static final BiomeBase m;
++    public static final BiomeBase n;
++    public static final BiomeBase o;
++    public static final BiomeBase p;
++    public static final BiomeBase q;
++    public static final BiomeBase r;
++    public static final BiomeBase s;
++    public static final BiomeBase t;
++    public static final BiomeBase u;
++    public static final BiomeBase v;
++    public static final BiomeBase w;
++    public static final BiomeBase x;
++    public static final BiomeBase y;
++    public static final BiomeBase z;
++    public static final BiomeBase A;
++    public static final BiomeBase B;
++    public static final BiomeBase C;
++    public static final BiomeBase D;
++    public static final BiomeBase E;
++    public static final BiomeBase F;
++    public static final BiomeBase G;
++    public static final BiomeBase H;
++    public static final BiomeBase I;
++    public static final BiomeBase J;
++    public static final BiomeBase K;
++    public static final BiomeBase L;
++    public static final BiomeBase M;
++    public static final BiomeBase N;
++    public static final BiomeBase O;
++    public static final BiomeBase P;
++    public static final BiomeBase Q;
++    public static final BiomeBase R;
++    public static final BiomeBase S;
++    public static final BiomeBase T;
++    public static final BiomeBase U;
++    public static final BiomeBase V;
++    public static final BiomeBase W;
++    public static final BiomeBase X;
++    public static final BiomeBase Y;
++    public static final BiomeBase Z;
++    public static final BiomeBase aa;
++    public static final BiomeBase ab;
++    public static final BiomeBase ac;
++    public static final BiomeBase ad;
++    public static final BiomeBase ae;
++    public static final BiomeBase af;
++    public static final BiomeBase ag;
++    public static final BiomeBase ah;
++    public static final BiomeBase ai;
++    public static final BiomeBase aj;
++    public static final BiomeBase ak;
++
++    private static BiomeBase a(String s) {
++        BiomeBase biomebase = (BiomeBase) BiomeBase.REGISTRY_ID.get(new MinecraftKey(s));
++
++        if (biomebase == null) {
++            throw new IllegalStateException("Invalid Biome requested: " + s);
++        } else {
++            return biomebase;
++        }
++    }
++
++    static {
++        if (!DispenserRegistry.a()) {
++            throw new RuntimeException("Accessed Biomes before Bootstrap!");
++        } else {
++            // Paper start - Biome Replacement from config
```

Comment:
.... no. Move this logic to the method right above.
```java
s = PaperConfig.biomeReplacement.getOrDefault(s, s);
```

Reply:
Sorry, move which logic to what method right above. Do you mean have a method that takes in a single string to prevent the same string from being used twice on each line?

Reply:
I see what your talking about, thanks I'll fix it along with the other requested fixes.